### ISSUE TITLE
SLIRP nw driver stops working quite often

### ISSUE THREAD

**Issue by gpsa** [abiosoft/colima#678]
### Description

After having some issues with [`gvproxy`](/abiosoft/colima/issues/466), I was suggested to use SLIRP, however I've been noticing that, during the day, it stops working and all the mounting points crash, although I see that the VM is still accessible (`colima ssh` works).
That's the same behaviour I noticed when using Lima directly, because I believe they use SLIRP by default.
The solution right now is to restart colima every time it happens, but is there anything that can be done to tackle this issue at Colima's domain, or would that be something more about Lima or even QEMU?

### Version

Colima Version: 0.5.4
Lima Version: 0.15.0
Qemu Version: 7.2.0

### Operating System

- [X] macOS Intel <= 12 (Monterrey)
- [ ] macOS Intel >= 13 (Ventura)
- [ ] macOS M1 <= 12 (Monterrey)
- [ ] macOS M1 >= 13 (Ventura)
- [ ] Linux

### Reproduction Steps

1. colima start # keep working on your containers and wait...that will eventually happen

### Expected behaviour

Mounting points to continue to work properly

**Comment by gpsa** [comment]
Unfortunately not, still waiting for some help, although I believe this has more to do with lima / QEMU because something similar happens when I'm just running `Lima` alone.

**Comment by fskaeh** [comment]
Hi @abiosoft, if I'm not mistaken, I believe Lima's latest 0.16 release should help with this issue: https://github.com/lima-vm/lima/pull/1383
But we might need a small update on Colima's side to handle this new value `user-v2` to replace slirp.

**Comment by abiosoft** [comment]
> Hi @abiosoft, if I'm not mistaken, I believe Lima's latest 0.16 release should help with this issue: [lima-vm/lima#1383](https://github.com/lima-vm/lima/pull/1383) But we might need a small update on Colima's side to handle this new value `user-v2` to replace slirp.

The main advantage of `user-v2` is for vm-vm communications.  e.g. a multi-node Kubernetes cluster or docker swarm.

**Comment by github-nicolas-stadler** [comment]
> > Hi @abiosoft, if I'm not mistaken, I believe Lima's latest 0.16 release should help with this issue: [lima-vm/lima#1383](https://github.com/lima-vm/lima/pull/1383) But we might need a small update on Colima's side to handle this new value `user-v2` to replace slirp.
> 
> The main advantage of `user-v2` is for vm-vm communications. e.g. a multi-node Kubernetes cluster or docker swarm.

Agreed, but as it stands VZ+Slirp is just broken (at least for me and some other people judging by the issues), so anything that lets us get rid off Slirp is very welcome.

**Comment by abiosoft** [comment]
> Agreed, but as it stands VZ+Slirp is just broken (at least for me and some other people judging by the issues), so anything that lets us get rid of Slirp is very welcome.

`user-v2` is being evaluated to see how best to integrate it in to Colima.
